java/ActigraphReader.java

```diff
@@ -80,7 +80,7 @@ public static void readG3TXEpochs(
             double sampleFreq = -1, accelerationScale = -1, _AccelerationMin, _AccelerationMax;
             long startDate = -1, stopDate = -1, firstSampleTime=-1;
             String serialNumber = "";
-            String infoTimeZone = "01:00:00"; // default to be UTC time difference
+            String infoTimeShift = "01:00:00"; // default to be UTC time difference
```

Comment:
For UTC shouldn't this be `"00:00:00"`?

Reply:
I think you are right. Let me test it out.

Reply:
Yeah `00:00:00` is actually correct after I've tested them out on different files :D Good catch!